package.json

```diff
@@ -49,11 +49,11 @@
     ]
   },
   "devDependencies": {
-    "@playwright/test": "^1.30.0",
+    "@playwright/test": "^1.32.2",
```

Comment:
I added this additional bump. Without it, playwright was complaining about a difference in versions.